google-cloud-clients/google-cloud-websecurityscanner/src/test/java/com/google/cloud/websecurityscanner/it/v1beta/VpcPositiveTest.java

```diff
@@ -0,0 +1,270 @@
+        return startedScanRun;
+      }
+
+      ScanRun stopScanRun() {
+        StopScanRunRequest stopScanRunRequest =
+            StopScanRunRequest.newBuilder().setName(startedScanRun.getName()).build();
+        ScanRun stoppedScanRun = wssClient.stopScanRun(stopScanRunRequest);
+        startedScanRun = null;
+        return stoppedScanRun;
+      }
+
+      @Override
+      public void close() {
+        if (startedScanRun != null) {
+          stopScanRun();
+          startedScanRun = null;
+        }
+
+        if (createdScanConfig != null) {
+          deleteScanConfig();
+          createdScanConfig = null;
+        }
+      }
+    }
+
+    try (TestResource testResource = new TestResource(wssClient, displayName)) {
+
+      ScanConfig responseScanConfig = testResource.createScanConfig();
+      assertEquals(
+          "Display name is response must be equal to display name in request",
+          displayName,
+          responseScanConfig.getDisplayName());
+
+      // Make sure that created scan config is present
+      GetScanConfigRequest scanConfigRequest =
+          GetScanConfigRequest.newBuilder().setName(responseScanConfig.getName()).build();
+      ScanConfig obtainedScanConfig = wssClient.getScanConfig(scanConfigRequest);
+      assertEquals(
+          "Scan-Config name must be equal",
+          responseScanConfig.getName(),
+          obtainedScanConfig.getName());
+
+      ListScanConfigsRequest lscRequest =
+          ListScanConfigsRequest.newBuilder().setParent(formattedParent).build();
+      boolean oneDisplayNameMatches = false;
+      for (ScanConfig sc : wssClient.listScanConfigs(lscRequest).iterateAll()) {
+        // since display name is never null so object equality can be used
+        if (displayName.equals(sc.getDisplayName())) {
+          oneDisplayNameMatches = true;
+          break;
+        }
+      }
+      assertTrue("One scan-config with " + displayName + " must be present", oneDisplayNameMatches);
+
+      ScanRun scanRunResponse = testResource.startScanRun();
+      assertTrue("Scan-run name must not be empty", isNotEmpty(scanRunResponse.getName()));
+
+      assertNotEquals(
+          "Scan-run state must not be KILLED",
+          ResultState.KILLED,
+          scanRunResponse.getResultState());
+
+      ScanRun stoppedScanRun = testResource.stopScanRun();
+      assertEquals(
+          "Result state must be killed", ResultState.KILLED, stoppedScanRun.getResultState());
+      testResource.deleteScanConfig();
+
+      // make sure that deleted scan-config do not exist
+      GetScanConfigRequest getScanConfigRequest =
+          GetScanConfigRequest.newBuilder().setName(responseScanConfig.getName()).build();
+      try {
+        ScanConfig deletedScanConfig = wssClient.getScanConfig(getScanConfigRequest);
+        fail("NotFoundException must be throw as scan-config must already be deleted");
+      } catch (NotFoundException nfe) {
```

Comment:
Please add a comment or rename the caught exception the "expected". https://google.github.io/styleguide/javaguide.html#s6.2-caught-exceptions

Reply:
Made this change.